### DISCUSSION TITLE
【委托】紧急！！！

### DISCUSSION THREAD

**Discussion by interknotazy** [share121/inter-knot#3351] [其他]
请立即上原神歪七七
任务奖励：uzi跳墙教学

**Comment by 0-Bai-Di** [comment]
第一我没有七七
第二 我有主播同款G
要不表演一个p18c跳枪？（doge）

**Comment by FGNGNGG** [comment]
已经歪了，uzi跳枪课程在哪领（doge）<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by lllliufeng** [comment]
还真歪了，快给我课程（
